teams/crate-maintainers.toml

```diff
@@ -25,3 +25,7 @@ orgs = ["rust-lang"]
 weight = -200
 name = "Crate maintainers"
 description = "Maintaining official rust-lang crates such as log, libc, cc, and more."
+zulip-stream = "t-libs/crates"
```

Comment:
👍 One day I hope this repo and `sync-team` could also just create these streams automatically...